packages/core/src/App/app.jsx

```diff
@@ -65,10 +66,20 @@ const AppWithoutTranslation = ({ root_store }) => {
 
     const handleResize = React.useCallback(() => {
         if (isTouchDevice() && (isMobile() || isTablet())) {
-            const is_android_device = mobileOSDetect() === 'Android';
-            const view_width = is_android_device ? screen.availWidth : window.innerWidth;
-            const view_height = is_android_device ? screen.availHeight : window.innerHeight;
+            let view_width, view_height;
             const el_landscape_blocker = document.getElementById('landscape_blocker');
+
+            if (mobileOSDetect() === 'Android') {
+                view_width = screen.availWidth;
+                view_height = screen.availHeight;
+            } else if (mobileOSDetect() === 'iOS' && isSafari()) {
+                view_width = window.outerWidth;
+                view_height = window.outerHeight;
```

Comment:
Check for outerWidth and outerHeight instead of innerWidth and innerHeight as the keyboard in iOS safari is changing the viewport, so need to check whole browser window, and not just the visible viewport